### PR TITLE
Prepare module for PS Accounts v4 with new Composer dependencies

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
* Remove hotfixes for PS Accounts
* Check EventBus as well as PS Account version
* [TECH] Replace library used to call PS Account module

## Check of EventBus version

image

## New status checks

image

Jira ticket related: https://forge.prestashop.com/browse/EMKTG-724

### CODE REVIEW

File: _dev/.storybook/translations.json [hunk 1]
```diff
@@ -31,9 +31,9 @@
         "unknownOnboardingError": "An unknown error occurred during onboarding process. Please reload and try again.",
         "reloadButton": "Reload",
 
-        "psAccountUpgradeNeededWarning": "The version of the module PrestaShop Account running on this shop (v{psAccountsVersion}) is older than the minimum required v{requiredPsAccountsVersion}.\n\nYou may use PrestaShop Facebook but some features (i.e product synchronization) won't be available until you upgrade PrestaShop Account.",
-        "psAccountUpgradeButton": "Upgrade PrestaShop Accounts",
-        "psAccountUpgradeDone": "PrestaShop Account has been successfully upgraded.",
+        "psEventBusUpgradeNeededWarning": "The version of the module PrestaShop Event Bus running on this shop (v{psEventBusVersion}) is older than the minimum required v{requiredPsEventBusVersion}.\n\nYou may use PrestaShop Facebook but some features (i.e product synchronization) won't be available until you upgrade PrestaShop Event Bus.",
+        "psEventBusUpgradeButton": "Upgrade PrestaShop Event Bus",
```
Comment (xGouley) [inline _dev/.storybook/translations.json:35]: Au niveau marketing, on met l'espace ?

File: _dev/src/views/configuration.vue [hunk 2]
```diff
@@ -47,8 +47,8 @@
         @onAdCampaignClick="onAdCampaignClick"
         class="m-3"
       />
-      <PsAccountsUpdateNeeded
-        v-if="needsPsAccountsUpgrade"
+      <PsEventBusUpdateNeeded
```
Comment (xGouley) [inline _dev/src/views/configuration.vue:50]: C'est un remplacement ? j'aurais cru qu'on aurait mis les 2 (accounts + eventbus).

File: classes/Config/Config.php [hunk 3]
```diff
@@ -24,7 +24,9 @@ class Config
 {
     const API_VERSION = 'v10.0';
 
-    const REQUIRED_PS_ACCOUNTS_VERSION = '3.2.2';
+    const COMPLIANT_PS_ACCOUNTS_VERSION = '3.0.0';
```
Comment (xGouley) [inline classes/Config/Config.php:27]: Sert à quoi ?

Comment (Quetzacoalt91) [reply classes/Config/Config.php:27]: For merchants upgrading, I don't want to see the module "broken" until they upgrade PS Account

**Review by margud — APPROVED** [review]

**Review by xGouley — APPROVED** [review]
If marius ok with PHP part, ok

**Comment by Quetzacoalt91** [comment]
Thanks for the review guys

**Pull request merged**